services/venice-router/src/main/java/com/linkedin/venice/router/VeniceRouterConfig.java

```diff
@@ -371,6 +375,12 @@ private void checkProperties(VeniceProperties props) {
      * should consider to use some number, which is proportional to the available cores.
      */
     routerIOWorkerCount = props.getInt(ROUTER_IO_WORKER_COUNT, 24);
+    perRouterStorageNodeThrottlerEnabled = props.getBoolean(ROUTER_PER_STORAGE_NODE_THROTTLER_ENABLED, true);
+    perStoreRouterQuotaBuffer = props.getDouble(ROUTER_PER_STORE_ROUTER_QUOTA_BUFFER, 3.0);
```

Comment:
So we assume one Router could receive 3x more traffic than other Routers?
If we have this config, does it mean the store read quota is bumped by 3x?

Reply:
the previously the per partition buffer was 10, which 10x, its still much less given the assumption is each router only handles 1/router_count of the quota.